src/pybind/mgr/dashboard/frontend/src/styles/defaults/_bootstrap-defaults.scss

```diff
@@ -75,17 +80,22 @@ $health-color-warning-800: #9d6d10 !default;
 
 // Chart colors.
 $chart-color-red: $red !default;
-$chart-color-blue: #06c !default;
-$chart-color-orange: #ef9234 !default;
 $chart-color-yellow: #f6d173 !default;
+$chart-color-translucent-red: $red-dim !default;
+$chart-color-blue: $blue !default;
+$chart-color-orange: $orange !default;
```

Comment:
One last thing I'm still wondering is why are we changing some colors.

I see the current color changes for the graphs:

Blue:
- prev: `#06c`
- now:`#007bff`

Orange:
- prev: `#ef9234`
- now: `#fd7e14 `

Reply:
@Pegonzal , these changes were suggested by @cloudbehl

Reply:
I don’t have color preference, but want to be consistent as we are using multiple color for blue/orange in multiple places.

Reply:
I agree. I think the previous chart colors were 1:1 mapped with patternfly colors. But for us it is good if we can keep consistency with the rest of our projects color palettes. So I am okay with these changes